ci/taos/plugins-good/pr-format-doxygen-tag.sh

```diff
@@ -163,12 +163,10 @@ function pr-format-doxygen-tag(){
                         break
                     else
                         echo "[DEBUG] $doxygen_lang: passed. file name: $curr_file, ($doxygen_rule_num)/$doxygen_rule_all tags are found."
-                        check_result="success"
                     fi
                     ;;
                 * )
                     echo "[DEBUG] ( $curr_file ) file is not source code with the text format."
-                    check_result="success"
```

Comment:
after removing this line, let's introduce a "skip" condition to handle not source codes.